Gemfile.lock

```diff
@@ -290,6 +290,7 @@ GEM
       sprockets (> 3.0)
       sprockets-rails
       tilt
+    seed_migration (1.2.3)
```

Comment:
Do we still need this gem? It appears that the rest of the client code isn't using it.

Reply:
Great catch - but I already fixed it in 41c4c9c